src/libsync/creds/httpcredentials.cpp

```diff
@@ -114,8 +114,8 @@ HttpCredentials::HttpCredentials(const QString &user, const QString &password, c
     , _ready(true)
     , _clientSslKey(key)
     , _clientSslCertificate(certificate)
-    , _keychainMigration(false)
-    , _retryOnKeyChainError(false)
+    , 
```

Comment:
Hm, there's a couple of weird formatting happening. Let's keep the clang-tidy fix aside for now. I need to fix the CI integration first anyway.